charts/rode-collector-tfsec/templates/deployment.yaml

```diff
@@ -30,14 +30,14 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default (printf "v%s" ($.Chart.AppVersion) ) }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
             - "--port={{ .Values.service.port }}"
-            - "--rode-host={{ .Values.rode.host }}"
-            {{- if .Values.rode.insecure }}
-            - "--rode-insecure={{ .Values.rode.insecure }}"
+            {{- if .Values.rode.auth.proxy.enabled }}
```

Comment:
Lost the `--rode-host` flag configuration here, seems to be the cause of the failing build